contribution/pull_request.md

```diff
@@ -1,10 +1,19 @@
-#### Submitting a pull request
+# Submitting a pull request
 ----
 
-Once you're satisfied with the commits you've made in your branch and have pushed it up, you're ready to submit a pull request!
+Once you're satisfied with the commits you've made in your branch and have pushed it up, you're ready to submit **a pull request**!
 
-Head back over to the parent repository at [https://github.com/codebuddies/codebuddies](https://github.com/codebuddies/codebuddies) and click on ["Pull Requests"](https://github.com/codebuddies/codebuddies/pulls).
+1. Head back over to the parent repository at [https://github.com/codebuddies/codebuddies](https://github.com/codebuddies/codebuddies)
+2. Click on ["Pull Requests"](https://github.com/codebuddies/codebuddies/pulls).
 
-Then, click the "new pull request" button and compare your branch with the `staging` branch on the CodeBuddies repo.
+3. Click the **new pull request** button and compare your branch with the `staging` branch on the CodeBuddies repo.
 
-Here are some tips on [how to create the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request).
\ No newline at end of file
+####Tips for Creating the Perfect Pull Request
+*The following is excerpted from Keavy's article on the Github blog, ["How to write the perfect pull request"](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)*:
+- Include the purpose of this Pull Request. For example: This is a spike to explore… This simplifies the display of… This fixes handling of…
+- Consider providing an overview of why the work is taking place (with any relevant links); don’t assume familiarity with the history.
+- Remember that anyone in the company could be reading this Pull Request, so the content and tone may inform people other than those taking part, now or later.
+- Be explicit about what feedback you want, if any: a quick pair of :eyes: on the code, discussion on the technical approach, critique on design, a review of copy.
```

Comment:
Also, the sentence seems to be incomplete, Maybe `if any:` should be `for example:`?